witchcraft/witchcraft.go

```diff
@@ -437,6 +440,16 @@ func (s *Server) WithDisableGoRuntimeMetrics() *Server {
 	return s
 }
 
+// WithMetricsBlacklist sets the metric blacklist to the provided set of metrics. The provided input is copied.
```

Comment:
Maybe it is worth documenting somehow that the metrics in the blacklist are those without the value suffix (`.count`, `.max`, etc)